### PR TITLE
Remove alexaforbusiness from v2

### PR DESCRIPTION
**Pull request by wty-Bryant**
alexaforbusiness has been deprecated for 1 year, this pr removes it from go v2

### CODE REVIEW

File: .changelog/41a4c0708d4b410392ea999a0a08129a.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "id": "41a4c070-8d4b-4103-92ea-999a0a08129a",
+    "type": "release",
+    "description": "remove deprecated alexaforbusiness from v2",
```
Comment (lucix-aws) [inline .changelog/41a4c0708d4b410392ea999a0a08129a.json:4]:
```suggestion
    "type": "feature",
    "description": "Remove deprecated `alexaforbusiness` service.",
```

Comment (lucix-aws) [reply .changelog/41a4c0708d4b410392ea999a0a08129a.json:4]: (feature is going to be the most applicable here and I believe we used it for previous deprecations)

**Review by lucix-aws — APPROVED** [review]
fixandship

File: .changelog/41a4c0708d4b410392ea999a0a08129a.json (hunk 1 above)
Comment (wty-Bryant) [reply .changelog/41a4c0708d4b410392ea999a0a08129a.json:4]: previous iotroborunner used release, anyway I will use feature in this and later remove PRs

**Pull request merged**